add bangolufsen documentation

mj23000 opened the pull request:
## Proposed change
Add documentation for the bangolufsen integration.

PR to add brand icons to core_integrations in brands repo: https://github.com/home-assistant/brands/pull/4605
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/93462
- Link to parent pull request in the Brands repository: already added https://github.com/home-assistant/brands/pull/3877 https://github.com/home-assistant/brands/pull/3879
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

mj23000 committed:
add bangolufsen documentation

mj23000 committed:
Simplify documentation for initial integration

mj23000 committed:
Merge branch 'next' into bangolufsen

klaasnicolaas commented on source/_integrations/bangolufsen.markdown:
```diff
@@ -0,0 +1,352 @@
+ha_category:
+  - Media Player
+  - Multimedia
+ha_release: 2023.6.0
```
```suggestion
ha_release: 2023.8
```

mj23000 committed:
Bump HA version

edenhaus reviewed (changes requested):
Please create a new PR in the brand's repo, where you copy the images from `custom_integrations` to `core_integrations`.

mj23000 committed:
Merge branch 'next' into bangolufsen

mj23000 committed:
Merge branch 'next' into bangolufsen

mj23000 committed:
Add beolink_set_relative_volume custom service
Bump HA release
tweaks

mj23000 committed:
Merge branch 'next' into bangolufsen

mj23000 commented:
> Please create a new PR in the brand's repo, where you copy the images from `custom_integrations` to `core_integrations`.
I have now done this.
https://github.com/home-assistant/brands/pull/4605

mj23000 committed:
Remove custom services

mj23000 committed:
Merge branch 'next' into bangolufsen

mj23000 committed:
Update documentation to match integration
Add Beolab 8 as supported device.

frenck committed:
2023.12: Beta release notes

frenck committed:
Sync codebase with docs for 2023.12

frenck committed:
Merge branch '2023.12' into rc

joostlek committed:
Fix small typos (#30056)

c0ffeeca7 committed:
Assist: add more examples of built-in sentences (#30054)

c0ffeeca7 committed:
V2C: fix link to their website (#30062)

closes #30057

c0ffeeca7 committed:
SmartThings: fix link (#30059)

joncar committed:
Update link to programming software. (#30061)

frenck committed:
Revert "add komfovent integration docs" (#30079)

jpbede committed:
Add removing custom polling interval as breaking change for ping (#30078)

c0ffeeca7 committed:
Moehlenhoff: fix dead link (#30086)

piitaya committed:
Add missing documentation for numeric input and preset mode features (#30070)

* Add missing documentation for number value and preset mode features

* Rename to numeric input

* Update features.markdown

Co-authored-by: Fletcher <fletcher.boyd@artifactory.org.au>

---------

Co-authored-by: Fletcher <fletcher.boyd@artifactory.org.au>

DroneDaddy committed:
Updated yolink.markdown to include Outdoor Motion Detector Sensor support (#30074)

I have tested the YoLink Outdoor Motion Detector Sensor (YS7805) and confirmed an automation worked within Home Assistant 2023.11.3.

rccoleman committed:
Fix cover state checks (#30083)

Two `value_template` templates were comparing the state of a cover entity to "on"/"off" in one place, and "open"/"closed" in another.  The latter are the correct state values.

FletcherAU committed:
Add note about google cred activation delay fixes #29430 (#30088)

koestler committed:
add go-iotdevie into the list of third-party tools supporting homeassistant discovery (#30023)

Co-authored-by: Jan Bouwhuis <jbouwh@users.noreply.github.com>

AJediIAm committed:
Update google.markdown (#30090)

When following the steps, the text "and click 'Go to credentials'" confused me since I was not offered such a button. The next step brought me to where I needed to go. 
The text "and click 'Go to credentials'" is not needed.

Shulyaka committed:
openai_conversation: Update documentation to reflect the newer models (#30096)

* openai_conversation: Update documentation to reflect the newer language models

* Missing period.

mhaowork committed:
Update mqtt.markdown to fix typo (#30100)

There's one too many right bracket causing the json parsor to fail.

Error log:

2023-12-01 23:45:07.964 WARNING (MainThread) [homeassistant.components.mqtt.discovery] Unable to parse JSON irrigation: '{"name": "Irrigation", "command_topic": "homeassistant/switch/irrigation/set", "state_topic": "homeassistant/switch/irrigation/state", "unique_id": "irr01ad", "device": {"identifiers": ["garden01ad"], "name": "Garden" }}}'

FletcherAU committed:
Clarify Slack member ID retrieval (#30091)

* Document member ID retrieval in house

* Normalise usage of member vs user

* Change instructions to ordered list

* Update source/_integrations/slack.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update slack.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

FletcherAU committed:
Clarify how Slack uses messages and blocks (#30089)

* Document Slack block limit per message

* Explain where blocks and message fields will be used by Slack

* Update example to clarify block/message usage

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

FletcherAU committed:
Move information on CT clamps to a template (#30107)

FletcherAU committed:
Fix anchor link to point to correct section of page (#30108)

FletcherAU committed:
Reword SSDP note and admin access explainer (#30111)

FletcherAU committed:
Link to suitable notification integrations (#30112)

FletcherAU committed:
Replace Google password auth instructions with expanded app password info (#30106)

* Replace Google password auth instructions with expanded app password info

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

nwithan8 committed:
Add missing forecast_template to configuration variables (#30119)

FletcherAU committed:
Move tts_service from example to configuration (#30113)

* Move tts_service from example to configuration

* Implement feedback by the author of this PR

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

starkillerOG committed:
Clearify independend Reolink push notification switch (#30123)

fyhertz committed:
Enhance Ubus Integration Documentation & Refine ACL Permissions (#30121)

* Enhance Ubus Integration Documentation & Refine ACL Permissions

The current documentation does not contain enough information to start using the ubus integration, and the current ACL config file gives more permission than is actually needed.

This change explains how to add a system user `hass` and documents the configuration of `/etc/config/rpcd` configuration file. Those steps are missing from the current configuration.

Credits for those valuable steps go to: https://github.com/kvj/hass_openwrt.

Further more, the current ACLs config file gives more permission than is actually needed to make the integration work.

* Tiny style tweaks

* Update source/_integrations/ubus.markdown

* Update source/_integrations/ubus.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Linux VirtuaBox installation: update (#30101)

c0ffeeca7 committed:
Assist: add troubleshooting on weather queries (#30073)

FletcherAU committed:
Abode intro tense and perspective correction (#30133)

FletcherAU committed:
Capitalise list items in Acmeda (#30134)

FletcherAU committed:
Fix word order in Actiontec intro (#30135)

GuySie committed:
Add Matter Geneva blog (#30082)

* Add Matter Geneva blog

* Rename 2023-11-30-nabu-cas-at-the-matter-member-meeting to 2023-11-30-nabu-cas-at-the-matter-member-meeting.markdown

* Update source/_posts/2023-11-30-nabu-cas-at-the-matter-member-meeting.markdown

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>

* Rename 2023-11-30-nabu-cas-at-the-matter-member-meeting.markdown to 2023-11-30-nabu-casa-at-the-matter-member-meeting.markdown

* Update and rename 2023-11-30-nabu-casa-at-the-matter-member-meeting.markdown to 2023-12-04-nabu-casa-at-the-matter-member-meeting.markdown

---------

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>

FletcherAU committed:
Fix capitalisation in Agent DVR (#30142)

FletcherAU committed:
Minor error correction and readability improvements for Namecheap's DNS (#30145)

FletcherAU committed:
Correct spelling errors and minor formatting issues in z-wave (#30143)

FletcherAU committed:
Restructure Adax instrurctions (#30136)

synesthesiam committed:
Start documentation of Wyoming satellites (#30099)

FletcherAU committed:
Slightly adjust word order of cups and remove duplicate note (#30147)

c0ffeeca7 committed:
Assist troubleshooting: add step how to use alias (#30155)

Commit:
Update thethingsnetwork.markdown (#30140)

* Update thethingsnetwork.markdown

Suggest MQTT for TTNv3

* Tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

tomlut committed:
Update sensor.rest.markdown (#30154)

c0ffeeca7 committed:
Installation on Windows, using VMware Workstation: update procedure (#30161)

* Installation on Windows: VMware update procedure

- update procedure to work with latest versioin of VMware workstation 17 Player
- Procedure is mostly based on / Credits to SimplySmartHouse on https://community.home-assistant.io/t/installing-home-assistant-on-vmware-player-17/570428
- Procedure was reproduced on a Windows 10 system, using a HAOS v 11.1 image.
- Implements feedback from #30128

* Fix typo, add troubleshooting step

* Fix typo

* Implement review feedback

FletcherAU committed:
Reword Adguard intro (#30137)

piitaya committed:
Add target humidity and humidifier toggle features (#30170)

* Add target humidity and humidifier toggle features

* Update source/dashboards/features.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Fix filtering issues (#30171)

c0ffeeca7 committed:
Authentication: update section on login (#30169)

jpbede committed:
Add hint about refresh interval of ping (#30076)

cagriy committed:
Add a new troubleshooting step for Something Went Wrong error (#30163)

Co-authored-by: Fletcher <me@fboyd.me>
Co-authored-by: Cagri Yucel <cagri@instant-ps.com>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

scop committed:
Add Huawei LTE button category and platform (#30150)

EAGrahamJr committed:
Update supported color modes to match code. (#30173)

frenck committed:
Merge branch 'current' into rc

frenck committed:
2023.12: Update social image

frenck committed:
2023.12: Correct blogpost date

frenck committed:
2023.12: Add release party live stream

frenck committed:
2023.12: Finalize backward-incompatible changes

frenck committed:
2023.12: Update changelog

frenck committed:
2023.12: Dev blogs + removed integrations

frenck committed:
2023.12: Finalize new and moved to UI integrations

Commit:
Remove UniFi Protect EA docs (#30176)

Haerteleric committed:
Update zha.markdown with CB3 info (#30178)

frenck committed:
2023.12: Finalize other noteworthy changes

frenck committed:
2023.12: Add backward-incompatible notice on scenes

frenck committed:
2023.12: Finalize re-import blueprints

frenck committed:
2023.12: Finalize new login

frenck committed:
2023.12: Tweaks

frenck committed:
2023.12: Finalize new thermostat card

frenck committed:
2023.12: Small re-ordering of sections

frenck committed:
2023.12: Add humidifier screenshot

frenck committed:
2023.12: Finalize numeric input tile card feature

frenck committed:
2023.12: Finalize default dashboard options

frenck committed:
2023.12: Process review comments

frenck committed:
2023.12: Finalize more todo

frenck committed:
2023.12: Finalize history dashboard

frenck committed:
2023.12: Add intro

frenck committed:
2023.12: Add missing image

frenck committed:
2023.12: Title & description

frenck committed:
2023.12: Gorgeous consistency is beautiful

frenck committed:
2023.12: Add URL to chapter 5 livestream

frenck committed:
Merge branch 'current' into rc

frenck committed:
Apply suggestions from code review

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

frenck committed:
2023.12: Change release title

higbyk committed:
Update todo.markdown (#30185)

Added missing closing backticks on the example yaml for todo.add_item

gcoan committed:
Update templating.markdown, typo correction (#30187)

aquette committed:
List Z-Wave.Me Razberry 1 (300 series) as supported (#30183)

Successfully tested on a Raspberry pi4 with the standard
/boot/config.txt additions

Signed-off-by: Arnaud Quette <arnaud.quette@free.fr>
Co-authored-by: Arnaud Quette <ArnaudQuette@eaton.com>

nohn committed:
set default ping interval to a more sane value of 15 seconds instead of 5 minutes. this fixes home-assistant/core#105163

jpbede committed:
Set default ping interval to 15 seconds instead 5 minutes (#30192)

jpbede committed:
Revert "Set default ping interval to 15 seconds instead 5 minutes"

jpbede committed:
Revert "Set default ping interval to 15 seconds instead 5 minutes" (#30194)

b-uwe committed:
Distributors (#30202)

* Add Cloudfree as a US distributor

* Clarify addressing US

GuySie committed:
Add HomeWizard blog (#30203)

dependabot[bot] committed:
Bump actions/stale from 8.0.0 to 9.0.0 (#30215)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

QuintinHill committed:
Add ble2mqtt to third-party tools supporting MQTT auto-discovery (#30214)

Signed-off-by: Quintin Hill <git@quintin.me.uk>

frenck committed:
Mark HomeWizard WWHA local (#30218)

frenck committed:
2023.12.1 (#30227)

piitaya committed:
Add dropdown style for hvac mode feature (#30222)

jpbede committed:
Update ping interval to 30 seconds (#30195)

klaasnicolaas committed:
Replace deprecated Ruby extension in Devcontainers (#30225)

dependabot[bot] committed:
Bump addressable from 2.8.5 to 2.8.6 (#30269)

frenck committed:
Fix search, upgrade Algolia docsearch v3 (#30241)

RobinvandenHurk committed:
Removed character (#30260)

:)

jcranky committed:
Update configuration.markdown (#30256)

jbouwh committed:
Fix mqtt documentation on device connections (#30257)

retrop7 committed:
Update tuya.markdown to include opening the project after creating it. (#30259)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

TaiPhamD committed:
fix container disable  jemalloc example (#30239)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

c0ffeeca7 committed:
Streamline style (#30272)

c0ffeeca7 committed:
Make it clearer where auth command has to be entered (#30205)

ottelo9 committed:
Update wolflink.markdown (#30274)

c0ffeeca7 committed:
Blueprints: add re-importing procedure (#30208)

ludeeus committed:
Show search modal over the nav-bar (#30278)

ulflulfl committed:
Update led_ble.markdown (#30287)

c0ffeeca7 committed:
Voice Assistant: 13 USD task: make some components reusable (#30286)

Co-authored-by: Keith Burzinski <kbx81x@gmail.com>

mikeodr committed:
Add documentation for `weather.get_forecasts` (#30189)

* Add documentation for `weather.get_forecasts`

This was missed as part of the 2023.12 release. Document
the new service and note the old is now deprecated.

* Tiny tweaks

* remove raw tags

* Apply suggestions from code review

* Update per review.

* Remove old deprecated service
* Put example to hourly and specify daily is the default
* Put example response in a details block.

* Update temlplate example to use get_forecasts.

* Update python script example for get_forecasts.

* Update source/_integrations/weather.markdown

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

* Update source/_integrations/weather.markdown

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

* Update examples for weather.get_forecasts

* Review update

Simplify template example for get_forecasts documentation.

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>
Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

milperks committed:
Update discord.markdown (#30293)

GuySie committed:
Add Silicon Labs blog (#30294)

* Add Silicon Labs blog

* Update Silicon Labs blog date

* Update source/_posts/2023-12-12-silicon-labs-official-partnership-nabu-casa.markdown

---------

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>

ludeeus committed:
Add redirect for latest ios releasenotes (#30300)

starkillerOG committed:
Add Reolink RLN12W NVR as supported (#30311)

c0ffeeca7 committed:
Add procedure on turning S3-BOX into a voice assistant (#30292)

c0ffeeca7 committed:
Voice: add separate troubleshooting section (#30280)

c0ffeeca7 committed:
S3-BOX: add procedure on customizing the status images (#30302)

Co-authored-by: Franck Nijhof <git@frenck.dev>

c0ffeeca7 committed:
Voice assistant cleanup (#30317)

c0ffeeca7 committed:
Assist section: Fix links (#30319)

frenck committed:
2023.12.2 (#30315)

balloob committed:
Fix URL for draw_assist redirect

synesthesiam committed:
Year of the Voice Chapter 5 blog post (#30284)

* Draft of blog post

* Add more text

* Move images, change title casing and social image

* Add videos and s3-box tutorial link

* Requested changes

* Shrink social image

* Update source/_posts/2023-12-13-year-of-the-voice-chapter-5.markdown

Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

* Update source/_posts/2023-12-13-year-of-the-voice-chapter-5.markdown

Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

* Update source/_posts/2023-12-13-year-of-the-voice-chapter-5.markdown

Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

* Update source/_posts/2023-12-13-year-of-the-voice-chapter-5.markdown

Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

* Update social again

* Update source/_posts/2023-12-13-year-of-the-voice-chapter-5.markdown

---------

Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

chrismaki committed:
Update thermostat.markdown (#30325)

Image link broken due to having wrong extension.

ih8gates committed:
Update fastdotcom.markdown (#30322)

Fixed then/than spelling error

c0ffeeca7 committed:
Remove deprecated service list_events in favor of get_events (#30196)

* Deprecate service list_events in favor of get_events

- The service `calendar.list_events` is deprecated in favor or `calendar.get_events`
- in 2023.12. Scheduled to be removed in 2024.6
- This PR adds a note that this service is deprecated
- The docs for `get_events` are a topic for a separate PR
- Unlike list_events, get_events supports multiple responses
- related parent PR in core: https://github.com/home-assistant/core/pull/102481

* remove newlines

* Remove dates to align with format of other deprecation notices

* WIP: add calendar.list_events

* Add example of get_event, remove list_event

* Fix exmaple in service-calls

stefanroelofs committed:
Update 2023-12-13-year-of-the-voice-chapter-5.markdown (#30332)

Hyperlink was broken. Wrong brackets were used.
Thanks.

x10 committed:
Fix next link on onboarding.markdown so it matches the navigation and doesn't skip over dashboards (#30226)

* Update onboarding.markdown to fix next page

* Getting started: fix order in side navigation, fix links

---------

Co-authored-by: c0ffeeca7 <k0ffeeka7@gmail.com>

c0ffeeca7 committed:
S3 box installation add connect button (#30329)

* S3-BOX installation: add connect button for ESP32-S3-BOX-3

- leave link to ESPHome projects website for other product variants

* Fix typo

c0ffeeca7 committed:
S3-Box voice assistant: Delete procedure instructing to delete the config (#30327)

* S3-Box voice assistant

- Delete procedure instructing to delete the config
- if the device had been used before

* Also remove procedure for ATOM ECHO

c0ffeeca7 committed:
Voice assistant: Fix content modularization (#30326)

* Voice assistant: Fix content modularization

- replace screenshot for a device-specific one
- add links
- fix typos

* Fix markdown

---------

Co-authored-by: Michael Hansen <hansen.mike@gmail.com>

balloob committed:
Reference box instead of box-3 (#30339)

* Reference box instead of box-3

* Fix links

frenck committed:
2023.12.3 (#30341)

atomicpapa committed:
Update s3-box-customize.markdown (#30351)

Replacing ```substitution:``` with ```substitutions:```

frenck committed:
Add missing entry to 2023.12 TOC (#30350)

c0ffeeca7 committed:
Authentication: remove screenshot showing disabled login screen (#30357)

jbouwh committed:
Cleanup old anchors to point to new config style (#30358)

tronikos committed:
Google Generative AI: Remove mention of waitlist and update a link (#30355)

moofur committed:
Update s3_box_voice_assistant.markdown (#30375)

Added instructions for entering flash mode on s3-box-3

farmio committed:
Fix KNX button address config type value (#30362)

MacDada committed:
splitting_configuration: fixed filename (#30363)

for the explanation to be consistent with the example

larhedse committed:
Update epson.markdown (#30369)

Added Epson 9400W projector to supported devices. And added comment about high/low lamp for color mode.

alanblake committed:
Update tplink.markdown (#30388)

jtwild committed:
Update wyoming.markdown (#30318)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Assist: local pipeline: add step to install openWakeWord (#30391)

* Assist: local pipeline: add step to install openWakeWord

- closes #30364

* Fix markdown

c0ffeeca7 committed:
Radio browser: add step how to start Radio Browser (#30389)

- closes #30373

c0ffeeca7 committed:
Getting started: Move Dashboard explainer (#30334)

* Getting started: Move Dashboard explainer

- from Terminology to Dashboard section

* Update source/getting-started/onboarding_dashboard.markdown

c0ffeeca7 committed:
Radio browser: fix content order (#30393)

c0ffeeca7 committed:
Assist: move noise/gain configuration troubleshooting topic (#30333)

c0ffeeca7 committed:
S3-BOX tutorial (#30397)

- add note that tutorial applies to all 3 product variants
- use S32-S3-BOX as generic term
- addresses feedback from #30343

mj23000 committed:
Merge branch 'home-assistant:current' into bangolufsen

jpbede commented:
Closing this as it was based of the wrong branch so it has tagged a lot of people.
Please open a new PR based on `next`
